src/measure_theory/vector_measure.lean

```diff
@@ -815,4 +815,65 @@ end
 
 end vector_measure
 
+namespace signed_measure
+
+open vector_measure
+
+open_locale measure_theory
+
+/-- The underlying function for `signed_measure.to_measure`. -/
+def to_measure' (s : signed_measure α)
+  (i : set α) (hi₁ : measurable_set i) (hi₂ : 0 ≤[i] s)
+  (j : set α) (hj₁ : measurable_set j) : ℝ≥0∞ :=
+@coe ℝ≥0 ℝ≥0∞ _ ⟨s.restrict i j,
+begin
+  rw [s.restrict_apply hi₁ hj₁, ← s.restrict_apply hi₁ hj₁],
```

Comment:
What's going on here?

Reply:
Haha, I have no idea how that `rw` became like that